ingest-app/src/cmr/ingest/api/subscriptions.clj

```diff
@@ -26,10 +34,18 @@
                  :subscription provider-id native-id body content-type headers)
         _ (lt-validation/validate-launchpad-token request-context)
         _ (api-core/verify-provider-exists request-context provider-id)
-        _ (acl/verify-ingest-management-permission
-            request-context :update :provider-object provider-id)
-        _ (acl/verify-subscription-management-permission
-            request-context :update :provider-object provider-id)
+        subscription-user (get (json/decode (:metadata concept)) "SubscriberId")
+        token-user (api-core/get-user-id request-context headers)
+        bypass-acls (subscription-user-matches-token? concept request-context headers)
+        _ (when-not bypass-acls
```

Comment:
I know you didn't start this convention, but I _really_ don't like this abuse of the `_` binding in this `let`. This feels very much like a code smell and I would love to see this cleaned up significantly

Reply:
I'm not sure about how to handle that section either. My first inclination is to end the let, hold an if and inside that do a new let.